app/dashboard/models.py

```diff
@@ -1922,6 +1922,23 @@ def get_sent_kudos(self):
 
         return kudos_transfers
 
+
+    @property
+    def get_average_star_rating(self):
+        """Returns the average star ratings (overall and individual topic)
+        for a particular user"""
+
+        feedbacks = FeedbackEntry.objects.filter(receiver_profile=self).all()
+        average_rating = {}
+        average_rating['overall'] = sum([feedback.rating for feedback in feedbacks]) / feedbacks.count()
+        # average_rating['code_quality_rating'] = sum([feedback.code_quality_rating for feedback in feedbacks]) / feedbacks.count()
```

Comment:
I assume these commented out lines are for upcoming features, right?

Reply:
Nevermind - I see they are in Octavio's PR 👍

Reply:
Since that PR is merged now, I will remove the comments now. :)